Jenkinsfile

```diff
@@ -131,7 +135,8 @@ sed -e 's#{BUILD_BRANCH}#${env.BRANCH_NAME}#g' \
             'KEEP_TEMP_FOLDER': (params.KEEP_TEMP_FOLDER ? 'yes' : '')
           ],
           extraFiles          : (params.BUILD_SMPE ? 'zowe-smpe.zip,fmid.zip,pd.htm,smpe-promote.tar,smpe-build-logs.pax.Z,rename-back.sh' : ''),
-          keepTempFolder      : params.KEEP_TEMP_FOLDER
+          keepTempFolder      : params.KEEP_TEMP_FOLDER,
+          buildDocker      : (params.BUILD_SMPE ? 'yes' : '')
```

Comment:
Not sure if this is variable is used or not, but seems wrong parameter. should be `params.BUILD_DOCKER` instead of `params.BUILD_SMPE`.

Reply:
you're right, bad merge from the code.